lib/tests/streamlit/delta_generator_test.py

```diff
@@ -200,6 +201,7 @@ def test_duplicate_widget_id_error(self):
         """Multiple widgets with the same generated key should report an error."""
         widgets = {
             "button": lambda key=None: st.button("", key=key),
+            "button_group": lambda key=None: st.feedback([1, 2], key=key),
```

Comment:
It's specified here as `button_group` and below with the key `feedback`. Is this intentional?

Reply:
yeah that's intentional based on how the test is written, because `button_group` is the underlying widget name and `st.feedback` is the streamlit command